JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/definitions/JavaMemberVisitor.java

```diff
@@ -79,7 +79,12 @@ public Void visitConstructor(ConstructorMember member) {
 		final Label constructorEnd = new Label();
 		final JavaWriter constructorWriter = new JavaWriter(context.logger, member.position, writer, method, definition);
 		constructorWriter.label(constructorStart);
-		CompilerUtils.tagConstructorParameters(context, javaModule, member.definition, member.header, isEnum);
```

Comment:
💭 Design question:  
Do we want everything in visitConstructor, or do we want to
- Have a separate Expression/Member/... for implicit constructors?
- Delegate somewhere else if it's an implicit constructor (how to prevent too much duplicate code, then)?